docs/projects.md

```diff
@@ -72,6 +72,7 @@ type AndroidProjectParams = {
   manifestPath?: string;
   packageName?: string;
   dependencyConfiguration?: string;
+  unstable_reactLegacyComponent?: string[] | null;
```

Comment:
How about adding Names postfix that indicates it's a list (plural) and is inline with internal representation?
```suggestion
  unstable_reactLegacyComponentNames?: string[] | null;
```